src/rendering/renderers/shared/texture/utils/crossOrigin.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Set cross origin based detecting the url and the crossorigin
+ * @param element - Element to apply crossOrigin
+ * @param url - URL to check
+ * @param crossorigin - Cross origin value to use
+ */
+export function crossOrigin(element: HTMLImageElement | HTMLVideoElement, url: string, crossorigin?: boolean | string): void
```

Comment:
not for this PR, but this looks like it should exist in assets package (maybe is does?)

Reply:
Looking at the LoadParsers I think it could probably be part of the `loadVideoTexture.ts` module which i'm working on. Will refactor.